src/components/order-service/order-service.component.ts

```diff
@@ -2,7 +2,8 @@ import {OrderServiceController} from './order-service.controller';
 
 export const orderService: angular.IComponentOptions = {
   bindings: {
-    service: '<'
+    serviceClass: '<',
```

Comment:
I assumed we're going to have a different component for that. They're vastly different.